core/executor/src/allocator.rs

```diff
@@ -78,35 +114,43 @@ impl FreeingBumpHeapAllocator {
 	}
 
 	/// Gets requested number of bytes to allocate and returns a pointer.
+	/// The minimum size which can be allocated is 8 bytes.
```

Comment:
This sounds like I cannot allocate like 4 or 0 bytes. But this is not the case, right?

Reply:
You can call `allocate(4)` or `allocate(0)`, but there will be a minimum bucket of 8 bytes allocated. So yeah I guess it's phrased ambiguously and removed the comment.

Reply:
That's better, but the best would be to actually write the unambigious comment.